src/net/fe/OptionsStage.java

```diff
@@ -81,6 +81,10 @@ public void execute() {
 			new OptionGroupEntity(140,
 				new OptionGroup("FOG COLOR", java.util.Arrays.asList("DAY", "NIGHT"), java.util.Arrays.asList("Makes fog-of-war appear white, washing-out the terrain", "Makes fog-of-war appear black, dimming the terrain")),
 				FEResources.getFogColor().ordinal()
+			),
+			new OptionGroupEntity(170,
+				new OptionGroup("ACTUAL ODDS", java.util.Arrays.asList("TRUE", "FALSE"), java.util.Arrays.asList("Display the actual odds of a hit/crit happening ", "Display the numerical result of the hit/crit calculation")),
+				FEResources.getActualOdds() ? 1 : 0
```

Comment:
The ordering in the list-of-values and ordering in the initially-selected-value-index needs to match; here the list of values has true first and false second, while the initially selected value has false first and true second.